charts/kubefed/charts/controllermanager/templates/deployments.yaml

```diff
@@ -22,10 +22,17 @@ spec:
       containers:
       - command:
         - /hyperfed/controller-manager
-        - "--v={{ .Values.logLevel }}"
-        image: "{{ .Values.repository }}/{{ .Values.image }}:{{ .Values.tag }}"
-        imagePullPolicy: "{{ .Values.imagePullPolicy }}"
+        - "--v={{ .Values.manager.logLevel }}"
```

Comment:
Wouldn't you need to change the to `controller`? As you have updated in `values.yaml` below.

Reply:
yes, I did now pushed the changes.